### ISSUE TITLE
Web version of 4.0.8 producing larger minified output

### ISSUE THREAD

**Issue by vijayaraghavanramanan** [jakubpawlowicz/clean-css#901]
The web version of clean-css at https://jakubpawlowicz.github.io/clean-css/  produces larger output than 4.0.7

I could be wrong but semantic merging and restructing rules seems to have worked better in 4.0.7. 

The output is 32KB ungzipped now versus 30.5KB earlier.

**Comment by jakubpawlowicz** [comment]
This is the v4.0.7 and v4.0.8 diff - https://github.com/jakubpawlowicz/clean-css/compare/v4.0.7...v4.0.8 which didn't change much. Would you be able to share your CSS publicly / privately for further investigation?

**Comment by vijayaraghavanramanan** [comment]
Probably not v4.0.7 but some recent version before that. 

Anyway, I have the three css files - one original and two minified files (different versions) at https://raw.githubusercontent.com/vijayaraghavanramanan/test/master/clean-css-test.zip

Thanks.

**Comment by jakubpawlowicz** [comment]
I've checked all 4.x versions and consistently got the following numbers:

* level 1 optimizations - 32846 bytes
* level 2 optimizations - 30031 bytes
* level 2 optimizations with semantic merging and reordering on - 28823 bytes

Have you specified any custom options in the web UI?

**Comment by vijayaraghavanramanan** [comment]
I have the same level 1 and level 2 optimization as yours. 

I also have IE7 compatibility. (I had chosen this in the earlier version as well). 

Yet I am getting 32846 bytes, exactly the one you have on top. Can't get eithr 30031 or 28823. 

Maybe because choosing IE7 compatibility is causing level 2 to be not picked up even if I choose them?

I can't see the option to specify custom options in the web UI.

**Comment by vijayaraghavanramanan** [comment]
Further to my previous comment, even if I leave the compatibility to default (IE10+), it's 32846 bytes. Should have been smaller ideally. So that by itself seems to be an issue.

**Comment by jakubpawlowicz** [comment]
I've just checked IE7+ compatibility and there's no regression in 4.x branch.
You can set options in web UI using gear icon in bottom right corner of drop area.

**Comment by vijayaraghavanramanan** [comment]
Yeah clicking and web gear and setting options is what I have been using. Sorry for the confusion -- I thought there are options even more than that. 

Anyway, without referencing to previous options, I can say the following: 

The file size is the same - 32,864 bytes irrespective of choosing any of the following options: 

- Default.
- Default Level 2 optimisations 
- IE7+ compatibility
- IE7+ compatility + default level 2 optimisations
- IE7+ compatibility + level 2 optimisations with semantic merging and reordering on. 
- Any other without IE7+ compatibility. 

I may have wrongly stated 4.0.7 earlier -- maybe it's an earlier 4 but at any rate, as above, my bug report is independent of previous versions now. 

Simply put, choosing options doesn't seem to make a difference.

**Comment by vijayaraghavanramanan** [comment]
Okay, this is embarrassing. I totally missed the "Apply" button!

Sorry! Happens!

**Comment by jakubpawlowicz** [comment]
No worries. If you think we could make it a little more obvious there's an "Apply" button or get rid of the button altogether, ideas are more than welcome.

**Comment by vijayaraghavanramanan** [comment]
Yeah, actually, not a bad idea to remove the "Apply" button. 

Plus saving the options using the browser's local storage.

**Comment by jakubpawlowicz** [comment]
I've just opened a relevant issues for that ^.

I'll close this one for now.
